### PR TITLE
recipe for npy-append-array

### PR DESCRIPTION
**Pull request by andife**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by xor2k** [comment]
Dear all, please name the package "npy-append-array" so that it is consistent with its Pip-name.

File: recipes/npy-append-array/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/xor2k/npy-append-array
+  summary: Create Numpy `.npy` files that are larger than the main memory
+  license: MIT
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/npy-append-array/meta.yaml:38]: Add comment:

```yaml
# License file manually packaged: See https://github.com/xor2k/npy-append-array/pull/1
license_file: LICENSE
```

I opened a PR to fix the need to manually package the license file in future in future releases (https://github.com/xor2k/npy-append-array/pull/1). The comment is a visual reminder to check the status of this in the future and to remove the manually packaged version when the license file is included in the source distribution.

**Review by synapticarbors — COMMENTED** [review]
One small request and then I'm happy to merge.

**Comment by xor2k** [comment]
Hello everybody, I'd like to release a new version of my package and also add a description, since it currently just says "Description None". Can I use the text from README.md somehow? Best from Berlin, Michael

**Comment by synapticarbors** [comment]
@xor2k - I believe a PR needs to be submitted to: https://github.com/conda-forge/npy-append-array-feedstock

where the `description` section of the `recipe/meta.yaml` is populated. You can see an example of one in:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L76-L80

**Comment by xor2k** [comment]
Okay, I've done everything, see

https://github.com/conda-forge/npy-append-array-feedstock/pull/1

**Pull request merged**